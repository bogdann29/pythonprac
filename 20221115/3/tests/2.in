alp = Alpha(c=10, z=2, a=42)
alp.e = 123
print(alp)
try:
	alp.f1 = 15
except AttributeError:
	print('error')
alq = AlphaQ(c=10, z=2, a=42)
alq.e = 123
print(alq)
try:
	alq.f1 = 15
except AttributeError:
	print('error')
