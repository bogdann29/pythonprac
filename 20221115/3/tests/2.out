a: 42, c: 10, e: 123, z: 2
error
a: 42, c: 10, e: 123, z: 2
error
